### DISCUSSION TITLE
Feature Request: Expose SQL table function `glob` as a scalar function.

### DISCUSSION THREAD

**Discussion by redviking1** [duckdb/duckdb#6370] [Ideas]
Please expose SQL table function `glob` as a scalar function so that I may pass the list of file path names that it produces, possibly sorted, as an argument to functions like `read_csv` and `read_parquet`.  For example:
```
SELECT *
FROM read_parquet(list_sort(glob('*.parquet')));
```

**Comment by Tishj** [comment]
I think this is a great idea, similar to how `range` can be used as either a scalar or a table function 👍
